static/i18n/en/app.json

```diff
@@ -1583,6 +1591,18 @@
     },
     "FeeTooHigh": {
       "title": "Network fees represent more than 10% of the amount"
+    },
+    "StellarWrongMemoFormat": {
+      "title": "Memo format is wrong"
+    },
+    "StellarMinimumBalanceWarning": {
+      "title": "A minimum balance is required to be kept in a Stellar account"
```

Comment:
to me the currency name should not be in the wording, it's not for translation

```
"A minimum balance is required to be kept in a {{currencyName}} account"
```


which interestingly makes this wording generic. to me we could drop Stellar prefix?